frontend/src/components/LineChart.vue

```diff
@@ -6,6 +6,22 @@
       max-width="100%"
       min-width="500px"
     >
+      <!-- Add Reset Zoom Icon -->
+      <v-tooltip>
+        <template #activator="{ props }">
+          <v-btn
+            v-bind="props"
+            color="input"
+            size="small"
+            @click="resetZoom()"
+            style="position: absolute; top: 120px; right: 45px; z-index: 10;"
```

Comment:
<img width="1792" alt="image" src="https://github.com/user-attachments/assets/cd88544a-31e1-464d-8158-aef3ecd164a6">
on LG screens, the button will float right off the plot

Reply:
You're correct. I tried to adjust it for large screens, but now it floats to left on a small screen. I mistakenly pushed these changes to the develop, assuming the branch protections were still in active. However, it seems this PR has already been merged. I will create a new PR to address and resolve this issue.